main.go

```diff
@@ -81,6 +82,11 @@ func main() {
 	}
 	ctrl.SetLogger(crzap.Logger(true))
 
+	err := metrics.NewMetricsExporter()
+	if err != nil {
+		setupLog.Error(err, "error initializing metrics exporter")
```

Comment:
if this is failing, i think we should exit, not just log it.

Reply:
thanks, updated